br/pitr-known-issues.md

```diff
@@ -58,3 +64,27 @@ Issue: [#13126](https://github.com/tikv/tikv/issues/13126)
 After a network partition failure in the cluster, the backup task cannot continue backing up logs. After a certain retry time, the task will be set to `ERROR` state. At this point, the backup task has stopped.
 
 To resolve this issue, you need to manually execute the `br log resume` command to resume the log backup task.
+
+## The actual storage space used by log backup is 2~3 times that of the incremental data displayed in the cluster monitoring metrics
+
+Issue: [#13306](https://github.com/tikv/tikv/issues/13306)
+
+This issue occurs because the monitoring metrics show the size of data after RocksDB compaction, while log backup stores KV data in a customized encoding format. The different storage methods lead to different data compression ratios, the difference of which is 2~3 times.
+
+Log backup does not store data the way RocksDB generates SST files, because the data generated during log backup might have a large range and a small content. In such cases, restoring data by ingesting SST files cannot improve the restoration performance.
+
+## The error `execute over region id` is returned when you perform PITR
+
+Issue: [#37207](https://github.com/pingcap/tidb/issues/37207)
+
+This issue occurs when you enable log backup during a full data import and afterwards perform a PITR to restore data at a time point during the data import.
+
+If there are a large number of hotspot writes for a long time (such as 24 hours) and if the OPS of each TiKV node is larger than 50k/s (you can view the metrics in Grafana: **TiKV-Details** -> **Backup Log** -> **Handle Event Rate**), there is a probability that this issue occurs.
```

Comment:
It reads like line 80 and line 82 provide two different reasons. I guess line 82 is a further description of line 80?

Reply:
Yes. Do you have any suggestion?

Reply:
How about merging them as one paragragh